src/modules/network/tcp/unabto_tcp.h

```diff
@@ -0,0 +1,38 @@
+#ifndef _UNABTO_TCP_H_
+#define _UNABTO_TCP_H_
+#include <unabto/unabto_context.h>
+
+#ifdef WIN32
+#include <basetsd.h>
+#include "windows/unabto_tcp_windows.h"
+#else
+#include "unix/unabto_tcp_unix.h"
+#include <unistd.h>
+#endif
+
+enum unabto_tcp_status {
+    UTS_OK,
+    UTS_WOULD_BLOCK,
+    UTS_FAILED,
+    UTS_CONNECTING
+};
+
+
+enum unabto_tcp_status unabto_tcp_write(struct unabto_tcp_socket* sock, const void* buf, const size_t len, size_t* written);
```

Comment:
typedef enum {
...
} unabto_tcp_status;
will probably look more like the other code